mbed/mbed.py

```diff
@@ -3011,6 +3011,51 @@ def help_():
     return parser.print_help()
 
 
+@subcommand('unittest',
+    dict(name='--skip-build', action='store_true', help='skip build step'),
+    dict(name='--skip-run', action='store_true', help='skip run step'),
+    dict(name='--clean', action='store_true', help='clean build data'),
```

Comment:
Can you also add the short version `-c`? This is present in the other commands that build code.